framework/include/interfaces/Coupleable.h

```diff
@@ -153,7 +159,7 @@ class Coupleable
    * @see Kernel::value
    */
   template <ComputeStage compute_stage>
-  const typename VariableValueType<compute_stage>::type &
+  const typename VariableValueType<compute_stage, Real>::type &
```

Comment:
it is a little weird that in the system classes the order is `type, compute_stage` but it is `compute_stage, type` here

Reply:
You're a little weird. But good point :smile: